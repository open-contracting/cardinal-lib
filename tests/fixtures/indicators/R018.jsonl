{"ocid": "F","tender": {"numberOfTenderers": 1, "procurementMethod": "open"}}
{"ocid": "D","tender": {"procurementMethod": "direct"}}
{"ocid": "N","tender": {"procurementMethod": "open"}}
