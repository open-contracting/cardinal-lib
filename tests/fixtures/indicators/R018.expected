{"OCID":{"F":{"R018":1.0}}}
